Scripts/extend.gd

```diff
@@ -0,0 +1,13 @@
+class_name extend
```

Comment:
Name could cause issues. Furthermore, remember to use PascalCase for class names. Would recommend a class name that better describes what it does.

Reply:
Additionally the class name should probably be something along the lines of UtilityFunctions. Up to you.